tests/bagatom/test_bagToXML.py

```diff
@@ -25,7 +25,7 @@ def bagxml(monkeypatch):
         bagatom.getOxum()
     """
     m = mock_open(read_data=BAGIT_CONTENTS)
-    monkeypatch.setattr('__builtin__.open', m)
+    monkeypatch.setattr('builtins.open', m)
```

Comment:
Looks like there are a lot of places where we patch `builtins.open`...

Reply:
Yes, I was looking at all the `builtins.open` usages. It was the way the codalib was written in python2. But, I can be more specific as you suggested in all the patches in python3. Or I can open an issue and fix this in master first and pull those changes here.